.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 TiKV Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test Bulk Load 
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install TiUP
+        run: curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: maven
+
+      - name: Checkout client-java
+        uses: actions/checkout@v2
+        with:
+          repository: tikv/client-java
+          path: client-java
+
+      - name: Build Client Java
+        run: mvn --file client-java/pom.xml clean install -DskipTests
+
+      - name: Run TiKV Cluster
+        run: /home/runner/.tiup/bin/tiup playground nightly --mode tikv-slim --kv 3 --monitor=false &
+
+      - name: Wait TiKV Cluster
```

Comment:
can we wait TiKV cluster online determinately?

Reply:
yes. we can do it in this issue https://github.com/tikv/migration/issues/19

Reply:
Do you mean we do it as a follow up? If so, this PR is ok to merge for me.